OpenRA.Mods.RA/Traits/Infiltration/Infiltrates.cs

```diff
@@ -58,6 +58,10 @@ bool IsValidOrder(Actor self, Order order)
 			if (order.ExtraData == 0 && order.TargetActor == null)
 				return false;
 
+			// Should not infiltrate own or allied
+			if (order.TargetActor.Owner.IsAlliedWith(self.Owner))
```

Comment:
That is still not an exploit then.
I see no reason not to allow it, as long as we disable it where we want to.